src/test/resources/hbz01.es.json

```diff
@@ -83,6 +79,10 @@
   } ],
   "hbzId" : "HT018454638",
   "id" : "http://lobid.org/resources/HT018454638#!",
+  "inCollection" : [ {
+    "id" : "http://lobid.org/resources/HT014176012#!",
+    "label" : "lobid Ressource"
+  } ],
```

Comment:
You loose a nice label here. If you want to keep it, the sidecar approach will not work out-of-the-box. Rather you will have to add it in the morph by setting a new rdf:subject.